spec/rubocop/cop/style/hash_syntax_spec.rb

```diff
@@ -1154,6 +1154,22 @@ def do_something
         RUBY
       end
 
+      it 'registers an offense when expression follows multiple assignments' do
+        expect_offense(<<~RUBY)
+          foo = foo = do_stuff arg, opt1: opt1,
```

Comment:
I think `foo = bar =` would read a bit better here.

Reply:
`price = price = generate_something_with args, and_shorthands:` read the production code. :D